hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java

```diff
@@ -55,10 +53,8 @@
 import org.apache.hadoop.metrics2.lib.DefaultMetricsSystem;
 import org.apache.hadoop.minikdc.MiniKdc;
 import org.apache.hadoop.net.NetUtils;
-import org.apache.hadoop.net.ServerSocketUtil;
 import org.apache.hadoop.ozone.client.CertificateClientTestImpl;
 import org.apache.hadoop.ozone.common.Storage;
-import org.apache.hadoop.ozone.om.OMConfigKeys;
```

Comment:
I think this import should not be removed. This seems to be causing compilation failure:
_[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /github/workspace/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java:[178,16] cannot find symbol
  symbol:   variable OMConfigKeys
  location: class org.apache.hadoop.ozone.TestSecureOzoneCluster
[ERROR] /github/workspace/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java:[179,16] cannot find symbol
  symbol:   variable OMConfigKeys
  location: class org.apache.hadoop.ozone.TestSecureOzoneCluster
[ERROR] /github/workspace/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java:[181,16] cannot find symbol
  symbol:   variable OMConfigKeys
  location: class org.apache.hadoop.ozone.TestSecureOzoneCluster
[ERROR] /github/workspace/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java:[183,16] cannot find symbol
  symbol:   variable OMConfigKeys
  location: class org.apache.hadoop.ozone.TestSecureOzoneCluster
[ERROR] /github/workspace/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java:[185,16] cannot find symbol
  symbol:   variable OMConfigKeys
  location: class org.apache.hadoop.ozone.TestSecureOzoneCluster_

Reply:
Thanks @dineshchitlangia, just noticed and fixed it.

A new usage of `OMConfigKeys` was added in the second merge from master.  I usually rebuild locally, but it seems I forgot it in this case.  Thankfully CI caught it.

In the end, I had to remove the new code, though, as it introduced failures in token-related test cases.